CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Sourcery CHANGELOG
 
 ---
+## Master
+
+### New Features
+- [PR](https://github.com/krzysztofzablocki/Sourcery/pull/897) Methods, Variables and Subscripts will be uniqued in both `[methods]`, `all[Methods]`, new accessor is introduced that doesn't get rid of duplicates `raw[Methods]`. The deduping process works by priority order (highest to lowest):
```

Comment:
```suggestion
- [PR](https://github.com/krzysztofzablocki/Sourcery/pull/897) Methods, Variables and Subscripts will be uniqued in both `[methods]` and `all[Methods]`. A new accessor is introduced that doesn't get rid of duplicates: `raw[Methods]`. The deduping process works by priority order (highest to lowest):
```

Reply:
Also, why are `all[Methods]` and `raw[Methods]` spelled with brackets here? I'm not familiar enough with Sourcery templating to get the reference.

Reply:
it just meant `allMethods` `allVariables` `allSubscripts`, better way to express that?

Reply:
Since there are only three, I'd probably just list them out, or if you want to be fancy, maybe `all[Methods/Variables/Subscripts]`, but honestly I think just listing them is better.